### ISSUE TITLE
Hello, may I ask if the model obtained by using other action recognition algorithms besides the following algorithms can be used in your visualization program?

### ISSUE THREAD

**Issue by ya-92626** [kennymckormick/pyskl#180]
Hello, may I ask if the model obtained by using other action recognition algorithms besides the following algorithms can be used in your visualization program?
口DG-STGCN (Arxiv) [MODELZOO]
口ST-GCN (AAAI 2018) [MODELZOO]
口ST-GCN++ (ACMMM 2022) [MODEL Z00]
口PoseConv3D (CVPR 2022 Oral) [MODELZOO]
口AAGCN (TIP) [MODELZOO]
口MS-G3D (CVPR 2020 Oral) [MODELZOO]
口CTR-GCN (ICCV 2021) [MODELZOO]

**Comment by kennymckormick** [comment]
Yes I think, as long as the demo code can successfully call the API to inference the recognition results, which corresponds to this line: https://github.com/kennymckormick/pyskl/blob/e36e39c365b1ac038da0f558c0e6a3de2c328cea/demo/demo_skeleton.py#L287

**Comment by ya-92626** [comment]
> Hello, may I ask if the model obtained by using other action recognition algorithms besides the following algorithms can be used in your visualization program?
Thank you for your reply. I will try, I hope to continue to communicate with you when I encounter problems in the future.
